### PR TITLE
cc6

### PR DESCRIPTION
**Pull request by baxance**
- [x] Top-level README “Table of Contents” is updated
 - [x] Feature tasks for this challenge are completed
 - [x] Unit tests written and passing
     - [x] “Happy Path” - Expected outcome
     - [ ] Expected failure
     - [ ] Edge Case (if applicable/obvious)
 - [x] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [x] Link to code
     - [x] Picture of whiteboard

### CODE REVIEW

File: java/challenges/lib/src/test/java/challenges/linkedList/LinkedListTest.java [hunk 1]
```diff
@@ -90,4 +90,76 @@ public void stringTest() {
     assertEquals(expected, actual);
   }
 
+  @Test //1 add a node to the end of the linked list
```
Comment (Mattpet26) [inline java/challenges/lib/src/test/java/challenges/linkedList/LinkedListTest.java:93]: Great work on your tests! Keep it up.
